hooks/boot/02-message-helpers.rb

```diff
@@ -39,6 +39,19 @@ def dev_new_install_message(kafo)
 
   def failure_message
     say "\n  <%= color('There were errors detected during install.', :bad) %>"
+
+    # Kafo 7.1.0 added puppet_report
+    if respond_to?(:puppet_report) && puppet_report&.failed_resources&.any?
+      say
+      puppet_report.failed_resources.each do |failed_resource|
+        say "#{failed_resource} failed. Logs:"
+        failed_resource.log_messages.each do |message|
+          say message
+        end
+      end
+      say
+    end
+
     say "  Please address the errors and re-run the installer to ensure the system is properly configured."
```

Comment:
```suggestion
    say "  Please address the errors and re-run the installer to ensure the system is configured properly."
```
I am not sure if I can contribute here but minor fine-tuning in UI text can lead to better UX. :smiley: